gengokit/middlewares/templates/endpoints.go

```diff
@@ -5,9 +5,20 @@ package middlewares
 
 import (
 	"github.com/go-kit/kit/endpoint"
+	svc "{{.ImportPath -}} /generated"
 )
 
-func InjectEndpointMiddlewares(in endpoint.Endpoint) endpoint.Endpoint {
+// WrapEndpoint will be called individually for all endpoints defined in
+// the service. Implement this with the middlewares you want applied to
+// every endpoint.
+func WrapEndpoint(in endpoint.Endpoint) endpoint.Endpoint {
```

Comment:
The names of `WrapEndpoint` and `WrapEndpoints` are a bit confusion to me. I thought they were switched until I read the comments. `WrapEndpoints` seems like the natural one that would wrap all of the Endpoint**s**.

Thoughts on `WrapAllEndpoints` and `WrapServiceEndpoints`? 

I know they are more verbose, but seeing "`WrapAllEndpoints`" really gets the message across to me without reading the doc string, but just "`WrapEndpoint`" sounds like it will only wrap one (singular) endpoint.